deploy/scripts/run_docker.sh

```diff
@@ -6,7 +6,8 @@ docker_image_with_tag="kindlingproject/agent-builder:latest"
 
 configs=(-v "$HOME/.config:/root/.config" \
   -v "$HOME/.ssh:/root/.ssh" \
-  -v "$HOME/.gitconfig:/root/.gitconfig") 
+  -v "$HOME/.docker:/root/.docker" \
```

Comment:
Unnecessary Mount

Reply:
I recall that this mount may break the CI workflow. Could you please also reset this file?